src/Application/FileSystem/RemovedAndAddedFilesProcessor.php

```diff
@@ -99,16 +99,16 @@ private function processAddedFiles(): void
 
     private function processDeletedFiles(): void
     {
-        foreach ($this->removedAndAddedFilesCollector->getRemovedFiles() as $smartFileInfo) {
-            $relativePath = $smartFileInfo->getRelativeFilePathFromDirectory(getcwd());
+        foreach ($this->removedAndAddedFilesCollector->getRemovedFiles() as $removedFiles) {
```

Comment:
Should be probably singular?
Also based on type[], it should be `SmartFileInfo`

![image](https://user-images.githubusercontent.com/924196/92316595-b66fe080-eff6-11ea-8322-eb6738f6f7c5.png)